.flake8

```diff
@@ -1,18 +1,17 @@
 [flake8]
-ignore = 
-    # line length is handled by black
-    E501,
-    # W503 & E203 are not PEP 8 compliant and conflicts with black
-    W503,
```

Comment:
Hey `W503` seems to be missing in the new flake8 config?

Reply:
We've replaced `ignore` with `extend-ignore`. `ignore` replaces the default ignore list with our own, `extend-ignore` adds on our own list to the default list. `W503` is in the default ignore list so we don't need to add it on ourselves.